src/templates/React InstantSearch Native/package.json

```diff
@@ -16,7 +16,7 @@
     "prop-types": "15.6.2",
     "react": "16.4.1",
     "react-instantsearch-native": "{{libraryVersion}}",
-    "react-native": "0.56.0"
+    "react-native": "0.55.4"
```

Comment:
React Native 0.56.0 was not compatible with the template (more accurately because of Expo I think).